### PR TITLE
create, edit, delete order

### PR DESCRIPTION
**Pull request by manhbld-0962**
==> Tittle: create, edit, delete order

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/18310

==> Attachment File:

### CODE REVIEW

**Comment by manhbld-0962** [comment]
ready

**Comment by manhbld-0962** [comment]
ready

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  def new
+    @order = Order.new
+  end
+
+  def create
+    @order = @booking.orders.new order_params
+    @order[:food_id] = params[:order][:food_id]
+
+    if @order.save
+      flash[:success] = t("messages.create_success", name: @order.food.titleize_name)
```
Comment (hungct-1121) [inline app/controllers/orders_controller.rb]: delegate đi em

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      render :new
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    if @order.update_attributes order_params
+      flash[:success] = t("messages.update_success", name: @order.food.titleize_name)
```
Comment (hungct-1121) [inline app/controllers/orders_controller.rb]: delegate đi

File: app/views/orders/_order.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<tr class="row" id = "<%= "order-#{order.id}" %>" >
+  <td><%= order.food.name %></td>
```
Comment (hungct-1121) [inline app/views/orders/_order.html.erb]: delegate đi nhé

**Comment by manhbld-0962** [comment]
ready

**Pull request closed without merging**